free-1.md

```diff
@@ -5,47 +5,63 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+ 
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+A DSL made for the daily intelecual who wants a better way to store personalbe information easily such as books/locations of owned books in a collection, reports, websites and other knowledge that is personalble to ones self. This DSL may incldude such a laguage that allows for easy input, recovery, and autaomitic storage and organization. This laguage would aim to help the need for those who struggle to keep track of their personalble knowlege. Right now people use a collection of accumaltive apps such as notepad, zoetero, passward apps, and other note based collections. If such a DSL existed it would have the capability for people to store all of this info securley within one place. It would allow for easy acces to the info as well as easy storage. 
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+How many times a day to you say hey thats interesting or hey I need to remember this for later or you struggle to remember where you stored a password/peice of info. This language would allow the user to instanly store whatever they needed into a data base through a hopfully simple language on their phone or computer. For instace userA wants to remeber where they stored a book in their book collection of 100+ books. UserA could use the language by simply typing the name of the book/author to recover the infomation that is stored about the book in the data base. Another example UserB is a teacher and wants to keep track of a particular website for their students to use in years to come. Inseated of putting the website in a google app of some kind she can use the language to keep track of it the database.
+
```

Comment:
These are very interesting use cases - the use of a database to store information about a physical object, as well as the location of a digital one. Is there a reason why you mention google specifically? Like, is there something about google products that you want to improve on for this project?